EEUniverse.Library/Client.cs

```diff
@@ -55,6 +60,15 @@ public async Task ConnectAsync()
             _messageReceiverThread.Start();
         }
 
+        /// <summary>
+        /// Sends a message to the server, synchronously.
```

Comment:
I think we can leave out ", synchronously" here
(the same applies for the others)